include/json/value.h

```diff
@@ -171,6 +171,9 @@ class JSON_API Value {
     CZString(ArrayIndex index);
     CZString(const char* cstr, DuplicationPolicy allocate);
     CZString(const CZString& other);
+#ifdef JSON_HAS_RVALUE_REFERENCES
+	CZString(CZString&& other);
```

Comment:
Re: indentations: Sure my bad
Re: noexcept: Unfortunately not all compilers that support rvalue references support noexcept